_layouts/single.html

```diff
@@ -42,6 +42,9 @@
             </nav>
           </aside>
         {% endif %}
+        {% if site.show_edit_on_github_link %}
```

Comment:
I think it should also be possible to override this per page, as you might have collections of single pages, that you want collaborators on, but others you do not.